.github/workflows/ci.yml

```diff
@@ -20,7 +20,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        env: ["isort", "docs", "py36-django22"]
+        python: ['3.6', '3.7', '3.8' ]
+        django: [ '2.2', '3.2' ]
```

Comment:
Additionally, please update the `name` of the job which now still references the `${{ matrix.env }}` which is no longer defined.